src/main/java/com/rogerr/custom/model/Product.java

```diff
@@ -0,0 +1,51 @@
+package com.rogerr.custom.model;
+
+public class Product {
+
+	private String id;
+	private String name;
+	private String description;
+	private double price;
```

Comment:
mostly avoid primitives in your Models because they can't be null.   Also, instead of doubles use BigDecimal in properties used for monetary values.